pkg/common-controller/snapshot_delete_test.go

```diff
@@ -130,159 +130,44 @@ var snapshotClasses = []*crdv1.VolumeSnapshotClass{
 func TestDeleteSync(t *testing.T) {
 	tests := []controllerTest{
 		{
-			name:              "1-1 - content with empty snapshot class is deleted if it is bound to a non-exist snapshot and also has a snapshot uid specified",
```

Comment:
Many of these tests are no longer required in this package, or are duplicates of the new ones I wrote below (3-1, 3-2, etc)